handlers/main.yml

```diff
@@ -1,11 +1,17 @@
 ---
 # handlers file for wildfly
 
+- name: systemd reload
+  systemd:
+    daemon_reload: yes
+  when: ansible_service_mgr == 'systemd'
+
 - name: restart wildfly
   service:
     name: wildfly
     state: restarted
 
+
```

Comment:
it's not needed

Reply:
But without this change the systemd service can`t start for the first time

Reply:
wat? I'm talking about this empty space